schema/rdl_schema_0.1.json

```
@@ -949,6 +918,61 @@
 				"TCY"
 			]
 		},
+		"Resource": {
+			"title": "Resource",
+			"type": "object",
+			"description": "Information about an individual version of the dataset.",
+			"required": [
+				"id",
+				"title",
+				"description",
+				"media_type",
+				"url"
+			],
+			"properties": {
+				"id": {
+					"title": "Resource identifier",
+					"type": "string",
+					"description": "An identifier for this resource. This should be internally unique."
+				},
+				"title": {
+					"title": "Resource title",
+					"type": "string",
+					"description": "A title for this resource."
+				},
+				"description": {
+					"title": "Resource description",
+					"type": "string",
+					"description":"Additional description of the resource not already covered by th eother resource fields." 
+				},
+				"media_type": {
+					"title": "Media type",
+					"type": "string",
+					"description": "The media type of the resource taken from the [IANA Media Types codelist](https://www.iana.org/assignments/media-types/media-types.xhtml). For example a custom binary file has media_type 'application/octet-stream. A geojson file has the media_type 'application/geo+json'."	
+				},
+				"spatial_resolution": {
+					"title": "Spatial resolution",
+					"type": "number",
+					"description": "The minimum spatial separation resolvable in the resource, measured in meters."
```

Disallow negative or zero resolutions.

```suggestion
					"description": "The minimum spatial separation resolvable in the resource, measured in meters.",
					"exclusiveMinimum": 0
```